lib/OpenQA/WebAPI/Controller/API/V1/Asset.pm

```diff
@@ -176,15 +175,21 @@ sub delete {
         $cond{type} = $args{type};
     }
     else {
-        return;
+        return undef;
+    }
+
+    my $asset = $self->schema->resultset("Assets")->find(\%cond);
+
+    return $self->render(json => {error => 'Asset not found.'}, status => 404) unless $asset;
+
```

Comment:
would you mind deleting these blank lines?

Reply:
ok, I deleted these blank lines.